src/js/components/quick-replies.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import { sendMessage } from '../services/conversation-service';
+import { getRGB, rgbToHsl } from '../utils/colors';
+
+export class QuickRepliesComponent extends Component {
+    static propTypes = {
+        accentColor: PropTypes.string,
+        isAccentColorDark: PropTypes.bool,
+        choices: PropTypes.array.isRequired
+    };
+
+    onReplyClick = ({text, payload}) => {
+        sendMessage(text, {
+            payload
+        });
+    };
+
+    render() {
+        const {choices, accentColor, isAccentColorDark} = this.props;
+
+        const buttonStyle = {};
+
+        if (accentColor) {
+            const rgb = getRGB(`#${accentColor}`);
+            const {h} = rgbToHsl(...rgb);
+            buttonStyle.backgroundColor = isAccentColorDark ? `hsl(${h}, 100%, 95%)` : `hsl(${h}, 100%, 98%)`;
```

Comment:
Can it happen where the accent color is the perfect blend and our manipulations actually do nothing to it, so the quick reply just shows as a solid button? i.e. the computed background color is identical or near-identical to the foreground color?